src/cswinrt/code_writers.h

```diff
@@ -5964,25 +5995,28 @@ public static Guid PIID = Vftbl.PIID;
 
         w.write(R"(% static class %
 {
+public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { % }));
```

Comment:
We should declare this internal for now so that that it is not an API surface change for non-exclusive interfaces.  And in your other PR, you can make it public.

Reply:
Done.